### PR TITLE
[PROGRAMMES-6710] Profile taglines

### PR DESCRIPTION
**Pull request by benolot**
Link: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6710

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Domain/Profile/profile.html.twig [hunk 1]
```diff
@@ -3,6 +3,9 @@
     <div class="programme__img 1/1 programme__img--hasimage">
         <div class="programme__img-box">
             {{ ds_shared('image', profile.getProfile().getImage(), 256, '') }}
+            {% if profile.getProfile().getTagline() %}
+                <strong class="programme__overlay-tagline br-box-higlight br-highlight-bg-onbg080 text--center centi text--shout">{{ profile.getProfile().getTagline() }}</strong>
```
Comment (cisco87) [inline src/Ds2013/Presenters/Domain/Profile/profile.html.twig]: You wouldn't like changing it to:

```<strong class="programme__overlay-tagline br-box-higlight br-highlight-bg-onbg080 text--center centi text--shout">
    {{ profile.getProfile().getTagline() }}
</strong> 
```

I guess it is a matter of preference, have a thought

**Comment by benolot** [comment]
Test this please

File: src/Ds2013/Presenters/Domain/Profile/profile.html.twig (hunk 1 above)
Comment (benolot) [reply src/Ds2013/Presenters/Domain/Profile/profile.html.twig]: Could do - it increases readability but adds whitespace (and bytes) to the markup. Not too much of a fan.

**Pull request merged**